src/clients/c++/experimental_api_v2/library/common.cc

```diff
@@ -228,7 +228,7 @@ InferInput::GetNext(
 
   while ((bufs_idx_ < bufs_.size()) && (size > 0)) {
     const size_t buf_byte_size = buf_byte_sizes_[bufs_idx_];
-    const size_t csz = std::min(buf_byte_size - buf_pos_, size);
+    const size_t csz = (std::min)(buf_byte_size - buf_pos_, size);
```

Comment:
Strange thing for windows to require... seems like a bug in their compiler

Reply:
In one of the standard headers for windows, min and max are defined as MACROS that takes two arguments as well. This causes a conflict between the macro and std::min function. Enclosing the (std::min)/(std::max) ensures the function is selected and not the macro.